add_to_app/android_view/flutter_module_using_plugin/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: An example Flutter module that uses a plugin.
 version: 1.0.0+1
 
 environment:
-  sdk: ^3.7.0-323.2.beta
+  sdk: ^3.7.0
```

Comment:
To make CI happy I'd go with `sdk: ^3.7.0-0`. This deals with the situation that `beta` frequently lags `stable` during the major revision update.

Reply:
Wow, nice trick, I had never thought about that.